third_party/flann.BUILD

```diff
@@ -0,0 +1,164 @@
+# Description:
+#    FLANN is a library for performing fast approximate nearest neighbor
+#    searches in high dimensional spaces. It contains a collection of
+#    algorithms we found to work best for nearest neighbor search and a system
+#    for automatically choosing the best algorithm and optimum parameters
+#    depending on the dataset.
+
+licenses(["notice"])  # BSD
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = [
+        "-layering_check",
+        "-parse_headers",
+    ],
+)
+
+FLANN_COPTS = [
+    "-fopenmp",
+    "-fexceptions",
+    "-Wno-non-virtual-dtor",
+    "-Wno-unknown-pragmas",
+    "-Wno-conversion",
+]
+
+BOOST_TARGETS = [
```

Comment:
Are all of these actually necessary? I'd like to make sure this is the minimal set that is actually required by flann.